epictrack-web/src/components/indigenousNation/IndigenousNationForm.tsx

```diff
@@ -85,9 +85,13 @@ export default function IndigenousNationForm({ ...props }) {
   }, [ctx.item]);
 
   const getStaffs = async () => {
-    const staffsResult = await staffService.getAll();
-    if (staffsResult.status === 200) {
-      setStaffs(staffsResult.data as never);
+    try {
+      const staffsResult = await staffService.getAll();
+      if (staffsResult.status === 200) {
+        setStaffs(staffsResult.data as Staff[]);
+      }
+    } catch (e) {
+      console.log(e);
```

Comment:
We have error notification component in place and a common error message constant. Please take a look at EventForm -> line number 343 for reference

Reply:
thank you ill do that now